src/Microsoft.PowerApps.TestEngine/Providers/ITestWebProvider.cs

```diff
@@ -13,16 +13,16 @@ namespace Microsoft.PowerApps.TestEngine.Providers
     /// </summary>
     public interface ITestWebProvider
     {
-        #nullable enable
+#nullable enable
         public ITestInfraFunctions? TestInfraFunctions { get; set; }
 
         public ISingleTestInstanceState? SingleTestInstanceState { get; set; }
 
         public ITestState? TestState { get; set; }
 
         public ITestProviderState? ProviderState { get; set; }
-        #nullable disable
 
+#nullable disable
```

Comment:
Why do we want to disable nullable? The whole thing should be nullable enabled, to get the best defence against null deref issues.

Reply:
rest dont add new properties